modules/seo-tools.php

```diff
@@ -14,5 +14,21 @@
  */
 
 include dirname( __FILE__ ) . '/seo-tools/jetpack-seo.php';
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+// Suppress SEO Tools output if any of the following plugins is active.
+$jetpack_seo_conflicting_plugins = array(
```

Comment:
Should we extend that list with other SEO plugins? Our existing list of Open Graph conflicting plugins include a few plugins that also add meta tags:
https://github.com/Automattic/jetpack/blob/4.9/class.jetpack.php#L200